loader_hub/notion/base.py

```diff
@@ -90,21 +90,32 @@ def read_page(self, page_id: str) -> str:
         """Read a page."""
         return self._read_block(page_id)
 
-    def query_database(
-        self, database_id: str, query_dict: Dict[str, Any] = {}
-    ) -> List[str]:
+    def query_database(self, database_id: str) -> List[str]:
         """Get all the pages from a Notion database."""
+        query_dict = {"page_size": 100}
+        pages = []
+
         res = requests.post(
             DATABASE_URL_TMPL.format(database_id=database_id),
             headers=self.headers,
             json=query_dict,
         )
-        data = res.json()
-        page_ids = []
-        for result in data["results"]:
-            page_id = result["id"]
-            page_ids.append(page_id)
+        if res.ok:
```

Comment:
I think it's better to just raise an error if the result is not ok instead of just returning an empty list.

Reply:
Good point thanks.

I implemented it using the raise_for_status() method

Eventually, we can add error handling: https://developers.notion.com/reference/errors

In this way, the user will know better what kind of error is being raised. If you'd like, I can do it next week.